extensions/barrage/src/main/java/io/deephaven/extensions/barrage/util/GrpcUtil.java

```diff
@@ -146,7 +146,7 @@ public static void safelyExecute(final FunctionalInterfaces.ThrowingRunnable<Exc
      *
      * @param runner the runnable to execute safely
      */
-    public static void safelyExecuteLocked(final Object lockedObject,
+    private static void safelyExecuteLocked(final Object lockedObject,
```

Comment:
Did you want to change the logging from error back to debug?

Reply:
Yes, but have to finish killing these calls first